SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -65,7 +72,7 @@ etc...
     };
     ```
 
-1. In your project's `PlayerController` header file(s), add the method declaration
+1. In your project's `PlayerController` header file(s), add the public method declaration
 `virtual void InitPlayerState() override;`.
 
 1. In your game's `PlayerController` class(es), add the following definition of the `InitPlayerState` function:
```

Comment:
Is there a reason we switch to "game" here? It's clearer to users to use consistent terms unless switching to game is more meaningful.

Reply:
Oh I see what you mean. Discussed with Giray and we're switching them all to `game's` since this is more accurate in this case.

Reply:
OK. In the context of the whole document, it uses "project" - I am following the whole document at the same time as this PR as we need to look at edits in context.
So we need to use a consistent term, unless it makes sense to flip terms. If we use different terms unnecessarily, it throws people off and makes them have to think about what they are doing and this makes the process seem hard and complex - we want it to seem smooth and easy!

Reply:
Here the `game's` makes sense. This `PlayerController` is a part of the `game` while the `game` is a member of the project as a whole.

Reply:
OK, if it makes sense.